packages/admin-panel/src/VizBuilderApp/views/Main.js

```diff
@@ -88,7 +93,9 @@ export const Main = () => {
         <Toolbar />
         <Container maxWidth="xl">
           <PreviewDataProvider>
-            <Panel />
+            <JsonEditorProvider>
```

Comment:
Added a new context here, in case we need more global state for JsonEditor in the future.